scalafix-tests/output/src/main/scala/test/NoExtendsApp.scala

```diff
@@ -0,0 +1,51 @@
+package test
+
+object Main {
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+trait Something
+object Main2 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main3 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main4 extends Something
```

Comment:
Right, I think adding a warning is our only option.
Honestly I don't know how common it is, but yes, let's warn the users.